docs/src/examples/components/Tooltip/Types/TooltipExample.tsx

```diff
@@ -3,7 +3,7 @@ import { Button, Tooltip } from '@stardust-ui/react'
 
 const TooltipExample = () => (
   <Tooltip content="Hello from tooltip!">
-    <Button icon="expand" />
+    <Button content="Click me!" />
```

Comment:
```suggestion
    <Button>Click me!</Button>
```

It's an example for children API...